src/udev/scsi_id/scsi_serial.c

```diff
@@ -126,6 +128,8 @@ static int sg_err_category_new(int scsi_status, int msg_status, int
         if (host_status) {
                 if (IN_SET(host_status, DID_NO_CONNECT, DID_BUS_BUSY, DID_TIME_OUT))
                         return SG_ERR_CAT_TIMEOUT;
+                if (IN_SET(host_status, DID_TRANSPORT_DISRUPTED))
```

Comment:
Please use `==` if there exists only matching entry.

Reply:
This is not addressed yet.

Reply:
> This is not addressed yet.

I did not notice it, updated